titiler/api/endpoints/mosaic.py

```diff
@@ -0,0 +1,399 @@
+            raise BadRequestError(
+                f"{mosaic.__class__.__name__} does not support update operations"
+            )
+        return mosaic.mosaic_def
+
+
+@router.get(
+    "/bounds",
+    response_model=cogBounds,
+    responses={200: {"description": "Return the bounds of the MosaicJSON"}},
+)
+def mosaicjson_bounds(resp: Response, mosaic_params: CommonMosaicParams = Depends()):
+    """Read MosaicJSON bounds"""
+    resp.headers["Cache-Control"] = "max-age=3600"
+    with MosaicBackend(mosaic_params.mosaic_path) as mosaic:
+        return {"bounds": mosaic.mosaic_def.bounds}
+
+
+@router.get("/info", response_model=cogInfo)
+def mosaicjson_info(resp: Response, mosaic_params: CommonMosaicParams = Depends()):
+    """
+    Read MosaicJSON info
+
+    Ref: https://github.com/developmentseed/cogeo-mosaic-tiler/blob/master/cogeo_mosaic_tiler/handlers/app.py#L164-L198
+    """
+    mosaic_path = mosaic_params.mosaic_path
+    with MosaicBackend(mosaic_path) as mosaic:
+        meta = mosaic.metadata
+        response = {
+            "bounds": meta["bounds"],
+            "center": meta["center"],
+            "maxzoom": meta["maxzoom"],
+            "minzoom": meta["minzoom"],
+            "name": mosaic_path,
+        }
+        if not mosaic_path.startswith("dynamodb://"):
+            mosaic_quadkeys = set(mosaic._quadkeys)
+            tile = mercantile.quadkey_to_tile(random.sample(mosaic_quadkeys, 1)[0])
+            assets = mosaic.tile(*tile)
+            asset_info = cog_info(resp, url=assets[0])
+            del asset_info["band_metadata"]
+            response["quadkeys"] = list(mosaic_quadkeys)
+            response = {**asset_info, **response}
+        return response
+
+
+@router.get(
+    "/tilejson.json",
+    response_model=TileJSON,
+    responses={200: {"description": "Return a tilejson"}},
+    response_model_exclude_none=True,
+)
+async def mosaic_tilejson(
+    request: Request,
+    response: Response,
+    tile_scale: int = Query(
+        1, gt=0, lt=4, description="Tile size scale. 1=256x256, 2=512x512..."
+    ),
+    tile_format: Optional[ImageType] = Query(
+        None, description="Output image type. Default is auto."
+    ),
+    mosaic_params: CommonMosaicParams = Depends(),
+):
+    """Create TileJSON"""
+    kwargs = {"z": "{z}", "x": "{x}", "y": "{y}", "scale": tile_scale}
+    if tile_format:
+        kwargs["format"] = tile_format
+    tile_url = request.url_for("mosaic_tile", **kwargs).replace("\\", "")
+    with MosaicBackend(mosaic_params.mosaic_path) as mosaic:
+        tjson = TileJSON(
+            name=mosaic.mosaic_def.name,
+            description=mosaic.mosaic_def.description,
+            attribution=mosaic.mosaic_def.attribution,
+            bounds=mosaic.mosaic_def.bounds,
+            center=mosaic.mosaic_def.center,
+            minzoom=mosaic.mosaic_def.minzoom,
+            maxzoom=mosaic.mosaic_def.maxzoom,
+            tiles=[tile_url],
+        )
```

Comment:
the mosaic backend has a `meta` propriety and I think we could do 

```
    with MosaicBackend(mosaic_params.mosaic_path) as mosaic:
         tjson = mosaic.meta.copy()
         tjson.update({"tiles": [tile_url]})
```

Reply:
Much better! https://github.com/developmentseed/titiler/pull/40/commits/38e14995c26202977313f4934d824962791d2394